catalog/systemd.ru.catalog

```diff
@@ -274,7 +274,7 @@ Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
 столбце файла /etc/fstab, либо в параметре Where= файла конфигурации юнита),
 не является пустым. Это никак не мешает монтированию, однако ранее находившиеся
 в нем файлы будут недоступны. Чтобы получить к ним доступ, вы можете вручную
-перемонтировать нижележащую файловую систему в другую точку.
+перемонтировать эту файловую систему в другую точку.
```

Comment:
@arvidjaar the change was already merged. Please open a new issue + PR if you want this to be changed again.